packages/data-point/src/reducer-helpers/constant/constant.js

```diff
@@ -0,0 +1,30 @@
+const cloneDeep = require("lodash/cloneDeep");
```

Comment:
the title of this PR is "Reducer object **tests**", but it's not clear how that's related to some of these code changes

Reply:
that's true, I can move them out was trying to keep things contained to what the changed needed to be, I'll send a Pr with only the constant and the other module

Reply:
please look at https://github.com/ViacomInc/data-point/pull/385 and approve first, afterwards this PR should be cleaner